src/services/utility-helpers.js

```diff
@@ -515,6 +507,15 @@ export function getLabelValue(label) {
   return 'Label could not be parsed';
 }
 
+export function decodeHTML(labelText) {
+  return labelText
+    .replace(/&amp;/g, '&')
+    .replace(/&lt;/g, '<')
+    .replace(/&gt;/g, '>')
+    .replace(/&quot;/g, '"')
+    .replace(/&apos;/g, "'");
+};
+
```

Comment:
It looks like we already have libraries (`entities`, `html-entities`, `he`, and `parse-entities`) installed that do html entity encoding/decoding.  What about replacing this with one of those or is a more limited decoding preferable?

Reply:
Replacing this with one of these sounds good to me.

Reply:
Replace it with `decode` from `html-entities`.